packages/cli/src/databases/entities/AuthUser.ts

```diff
@@ -1,19 +1,19 @@
-import { Column, Entity, PrimaryColumn } from '@n8n/typeorm';
+import omit from 'lodash/omit';
+import { Column, Entity } from '@n8n/typeorm';
+import { User } from './User';
 
 @Entity({ name: 'user' })
-export class AuthUser {
-	@PrimaryColumn({ type: 'uuid', update: false })
-	id: string;
-
-	@Column({ type: String, update: false })
-	email: string;
-
-	@Column({ type: Boolean, default: false })
-	mfaEnabled: boolean;
-
+export class AuthUser extends User {
 	@Column({ type: String, nullable: true })
 	mfaSecret?: string | null;
 
 	@Column({ type: 'simple-array', default: '' })
 	mfaRecoveryCodes: string[];
+
+	override toJSON() {
+		return omit(super.toJSON(), ['mfaSecret', 'mfaRecoveryCodes']) as Omit<
+			this,
+			'password' | 'apiKey'
```

Comment:
Don't we need `'mfaSecret' | 'mfaRecoveryCodes'` here?

Reply:
this is an unfortunate force-typing to satisfy the type-checker complaining about `toJSON` not matching the signature in the base class. 
I'll try to replace this with something else that makes a lot more sense.